### PR TITLE
Documented example for each method

### PR DESCRIPTION
**Pull request by jaydenC88**

### CODE REVIEW

**Review by moealani — COMMENTED** [review]
Do you think using dashes instead of underscores is better so we can be consistent with the rest of the documentation? Other than that great job 👍

**Comment by jaydenC88** [comment]
@moealani you are right, I didn't notice the difference, thanks

File: CAM2CameraDatabaseAPIClient/client.py [hunk 1]
```diff
@@ -204,6 +212,11 @@ def update_owner(self, clientID, owner):
         str
             Success message.
 
+        Example
+        -------
+
+            client.update_owner(self.clientID, 'testowner')
```
Comment (mush-zhang) [inline CAM2CameraDatabaseAPIClient/client.py]: usually this method will be used to: `webUI_client.update_owner(userClientID, 'userClientOwner')` We might want to add more descriptive explanation and use-case scenario.

**Review by mush-zhang — CHANGES_REQUESTED** [review]
I have left one comment on the update_owner method. This would apply to other methods, especially those methods that are only accessible with webUI permission.

**Review by mush-zhang — APPROVED** [review]

**Review by Sripathm2 — APPROVED** [review]

**Review by Mrjimjr — APPROVED** [review]

**Pull request merged**